src/components/dashboard/PendingTxs/PendingTxsList.tsx

```diff
@@ -43,12 +42,22 @@ const PendingTxsList = ({ size = 4 }: { size?: number }): ReactElement | null =>
   const router = useRouter()
   const url = `${AppRoutes.transactions.queue}?safe=${router.query.safe}`
 
-  const queuedTxns: Transaction[] = (page?.results || []).filter(isTransactionListItem)
+  const queuedTxns = useMemo(() => {
+    return groupConflictingTxs(page?.results || [])
+      .map((group) => {
+        if (Array.isArray(group)) {
+          const latestTx = group.reduce((prev, curr) => {
+            return curr.transaction.timestamp > prev.transaction.timestamp ? curr : prev
+          })
```

Comment:
Aren't they already sorted by date by the backend?

In any case, `group.sort((a, b) => a.timestamp - b.timestamp).pop()` would be probably much simpler?